framework/python/src/core/testrun.py

```diff
@@ -60,7 +60,8 @@
 DEVICE_TEST_MODULES = 'test_modules'
 MAX_DEVICE_REPORTS_KEY = 'max_device_reports'
 
-VERSION = '1.2.2'
+# Does not need to be updated all the time, just a fallback
+VERSION = '1.3-alpha'
 

```

Comment:
Is there a reason we can't just resolve this from the control file directly instead of potentially having a bad fallback value?

Reply:
Removed the fallback